### DISCUSSION TITLE
Asterisk package possible?

### DISCUSSION THREAD

**Discussion by cracyfloyd** [buanet/ioBroker.docker#217] [Questions & Answers / Fragen & Antworten] [answered]
Is it possible to install the Asterisk package. I try but the docker dont find the package to install.
And an other question is. It is possible to use the yahka adapter in Bridge mode?

**Comment by buanet** [comment, marked as answer]
> Is it possible to install the Asterisk package. I try but the docker dont find the package to install.

You can install any package available in the matching packages repository. The base image in V6 is debian bullseye, so if there is a package in the debian bullseye package repository you are able to install it. 
But what is the reason to install asterisk in a ioBroker Docker container??? It is not recommended adding other software to a container. Use a dedicated container for your asterisk.   

> It is possible to use the yahka adapter in Bridge mode?

No. As the device detection is not working in bridge mode you still can run the adapter but you are not able to find/ connect.

Regards,
André
